### ISSUE TITLE
SecurityOptions type does not match type validations defined in the getSecurity method.

### ISSUE THREAD

**Issue by rychkog** [filestack/filestack-js#185]
Here is what we have in the documentation:
https://github.com/filestack/filestack-js/blob/7cd320a/src/lib/api/security.ts#L30
https://filestack.github.io/filestack-js/interfaces/securityoptions.html

It says that `call` is an array and accepts various permissions

But in reality when you pass an array of permissions like

```javascript
{ 
  call: ['read', 'write'],
  expiry: ...
}
```

`getSecurity` throws

```
Debug: internal, error 
    Error: Invalid value [
  "read",
  "write"
] supplied to "pick" | "read" | "stat" | "write" | "writeUrl" | "store" | "convert" | "remove" | "exif"
```

Because of incorrect validation in here
https://github.com/filestack/filestack-js/blob/7cd320a/src/lib/api/security.ts#L56

Do you consider creating a test suite for the API?

**Comment by pcholuj** [comment]
Hi. New release will fix this bug. It will be released in few days

**Comment by pcholuj** [comment]
Fixed in newest release 1.10.0
